mods/ts/rules/nod-vehicles.yaml

```
@@ -547,3 +547,88 @@ STNK:
 	Explodes@ELITE:
 		RequiresCondition: rank-elite
 		Weapon: UnitExplode
+
+SGEN:
+	Inherits: ^Tank
+	Inherits@VOXELS: ^VoxelActor
+	Inherits@selection: ^SelectableSupportUnit
+	-AppearsOnRadar:
+	Buildable:
+		Queue: Vehicle
+		BuildPaletteOrder: 130
+		Prerequisites: ~naweap, nastlh
+		Description: Can cloak units\nwhen deployed.\n  Unarmed
+	Valued:
+		Cost: 1600
+	Tooltip:
+		Name: Mobile Stealth Generator
+		RequiresCondition: !deployed
+	Tooltip@DEPLOYED:
+		Name: Mobile Stealth Generator (deployed)
+		RequiresCondition: deployed
+	Selectable:
+		Bounds: 1206, 1768, 0, -603
+	Health:
+		HP: 20000
+	Armor:
+		Type: Light
+	Voiced:
+		VoiceSet: Spy
+	Mobile:
+		Speed: 85
+		TurnSpeed: 20
+		ImmovableCondition: !undeployed
+		RequireForceMoveCondition: !undeployed
+	RevealsShroud:
+		RequiresCondition: !inside-tunnel && undeployed
+		Range: 5c0
+		MaxHeightDelta: 3
+	RevealsShroud@DEPLOYED:
+		RequiresCondition: inside-tunnel || deployed
+		Range: 6c0
+		MaxHeightDelta: 3
+	WithMakeAnimation:
+	GrantCondition@PREVIEWWORKAROUND:
+		Condition: real-actor
+	RenderSprites:
+		Image: mstl
+		PlayerPalette: playertem
+	GrantConditionOnDeploy:
+		PauseOnCondition: empdisable || being-captured
+		DeployedCondition: deployed
+		UndeployedCondition: undeployed
+		UndeployOnMove: true
+		UndeployOnPickup: true
+		Facing: 384
+		AllowedTerrainTypes: Clear, Road, DirtRoad, Rough
+		DeploySounds: place2.aud
+		UndeploySounds: clicky1.aud
+		Voice: Move
+	EntersTunnels:
+		RequireForceMoveCondition: !undeployed && !empdisable
```

```suggestion
		RequireForceMoveCondition: !undeployed
```

That's not how it works. The `empdisable` should on `RequiresCondition` or `PauseOnCondition`, though it seems that none of these traits are conditional. Repairable will become conditional with the docking refactor, but others won't

---

You explicitly asked for this in the comment before...

---

You are now making me apply changes and revert them again.

---

I didn't check to see wether they were conditional